troubleshooting/known_issues/barebones_boot_test_failure.md

```diff
@@ -0,0 +1,58 @@
+cray.barebones-boot-test: INFO     Created CFS session 'example-session'
+cray.barebones-boot-test: INFO     Waiting for CFS session 'example-session' to complete
+cray.barebones-boot-test: INFO     CFS session 'example-session' 'status' field is 'pending'
+cray.barebones-boot-test: INFO     CFS session 'example-session' 'status' field is now 'running'
+cray.barebones-boot-test: INFO     CFS session 'example-session' 'status' field is now 'complete'
+cray.barebones-boot-test: INFO     CFS session 'example-session' 'succeeded' field is now 'true'
+cray.barebones-boot-test: INFO     CFS session 'example-session' completed successfully
+cray.barebones-boot-test: INFO     Created IMS image '92045173-0735-4d3c-99fc-a198aa72bb29'
+cray.barebones-boot-test: INFO     CFS session 'example-session' created customized IMS image '92045173-0735-4d3c-99fc-a198aa72bb29'
+cray.barebones-boot-test: INFO     Creating BOS session template 'example-session'
+cray.barebones-boot-test: INFO     Created BOS session template 'example-session'
+cray.barebones-boot-test: INFO     Creating BOS session 'example-session' with BOS session template 'example-session' on node 'x3000c0s17b3n0'
+cray.barebones-boot-test: INFO     Created BOS session 'example-session'
+cray.barebones-boot-test: INFO     Waiting for BOS session 'example-session' to complete
+cray.barebones-boot-test: INFO     BOS session 'example-session' 'status' field is 'pending'
+cray.barebones-boot-test: INFO     BOS session 'example-session' 'status' field is now 'running'
+cray.barebones-boot-test: ERROR    BOS session 'example-session' has not completed even after 30 minutes
+cray.barebones-boot-test: ERROR    Failure of barebones image boot test.
+cray.barebones-boot-test: INFO     For troubleshooting information and manual steps, see https://github.com/Cray-HPE/docs-csm/blob/release/1.6/troubleshooting/cms_barebones_image_boot.md 
+```
+
+The test failure is attributed to incorrect network configuration, which causes the node to appear as though it has not booted. In most cases, the node has successfully booted but is inaccessible via the network.
+
+## Fix
+
+There is no fix for the barebones image in CSM 1.6.0. The problem exists in the barebones image as indicated above and has been patched in CSM 1.6.1 and later.
+
+If necessary, access to the node may be possible through the use of Cray Console.
+
+1. Find the node xname in the test output.
+
+    Look for text similar to the following in the test output:
+
+    ```text
+    Found compute node 'x3000c0s17b3n0' with architecture 'x86'
+    ```
+
+1. Follow the [Log in to a Node Using ConMan](../../operations/conman/Log_in_to_a_Node_Using_ConMan.md) procedure, using the xname identified in the previous step.
+
+For additional troubleshooting information see [CSM Troubleshooting Information](../troubleshooting/README.md)
```

Comment:
```suggestion
```
I see my other comment was ambiguous. What I meant was that we should update the troubleshooting README page with a link to this one, since it has a list of all the known issues. Sorry for the confusion.